src/CodeGen.cpp

```diff
@@ -174,7 +174,7 @@ bool CodeGen::llvm_NVPTX_enabled = false;
 bool CodeGen::llvm_Mips_enabled = false;
 
 void CodeGen::compile(Stmt stmt, string name,
-                      const vector<Argument> &args,
+                      const ArgInfo &arg_info,
```

Comment:
> Re: the merge with other branches, adding an argument to compile() rather than simply changing it seems like it's going to be a comparable amount of work either way.

It looked like the only change in the body of several affected functions was just to change 'args' to 'arg_info.args'. Anyways, we shouldn't let that determine the design, that only motivated me from being on the fence to actually say something.

BTW, I'm pretty sure that the number of pass-through calls will go down significantly with the codegen_compile_refactor branch, and then completely disappear with the modules branch.